torchvision/models/squeezenet.py

```diff
@@ -36,10 +34,10 @@ def forward(self, x):
 
 class SqueezeNet(nn.Module):
 
-    def __init__(self, version='1_0', num_classes=1000):
+    def __init__(self, arch, num_classes=1000):
         super(SqueezeNet, self).__init__()
         self.num_classes = num_classes
-        if version == '1_0':
+        if arch == 'squeezenet1_0':
```

Comment:
This is a BC-breaking change.

It's fine to do it for `ShuffleNetV2` because it hasn't been in any release yet, but SqueezeNet has been around for a while and this will break user code who relied on `SqueezeNet`.